graph-rag/app.py

```diff
@@ -1,14 +1,21 @@
 from fastapi import FastAPI, Request
 import os, re
 from fastapi.responses import JSONResponse
-from neo4j import GraphDatabase, AsyncGraphDatabase
+from neo4j import GraphDatabase
 from langchain_community.embeddings import OllamaEmbeddings
 from langchain_community.chat_models import ChatOllama
+from langchain_community.embeddings
 
-from prompts import PROMPT_QWEN
+from langchain_community.embeddings.openai import OpenAIEmbeddings
+from langchain_community.chat_models.openai import ChatOpenAI
+os.environ['OPENAI_API_KEY'] = '***'
```

Comment:
add a comment to replace this

Reply:
also, given this is a public doc, the imports and variables are all over the place. Can we follow some best practices for organizing the files?